### ISSUE TITLE
Adding Default Labels on New Issue

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/ubiquibot#325]
A nice config to have would be to allow repository maintainers to specify the default labels that the bot should automatically add for any newly created issue if there are no labels. 

For example:

When I create new issues, I do several things before pressing the "Submit new issue" button generally speaking: 1. Write the issue title 2. Write the issue body 3. Add labels.

**"3." Means that it is possible to create a new issue with no labels (or with the proper `Priority: ` and `Time: ` labels.)**

It would enhance the bounty issuing experience if we can set a repository default in `ubiquibot-config.yml` to automatically add specific labels on issues opened without any labels. In this instance, I would be interested to have the bot automatically add `Priority: 0 (Normal)` and `Time: <1 Hour` (or `Time: <1 Day`) so that I can rapid fire issuing bounties.

**Comment by rondDev** [comment]
Would this involve creating custom types for default config or could WideConfig be extended?

If WideConfig should be extended, would it make sense to use Priority as `name` and Time as `target`?

Realistically, it should be extended with types, but I am not that talented in TypeScript, lol

**Comment by pavlovcik** [comment]
I don't understand your question. Maybe @0xcodercrane can provide input

**Comment by rondDev** [comment]
Basically avoid implementing whole new handling for Default Labels

![example](https://rond.cc/u/z8Xi4w8zg8Yj.png)

**Comment by pavlovcik** [comment]
I'm not that familiar with the code so I'll wait on a team member to provide input.

**Comment by 0xcodercrane** [comment]
To clarify things it from the origin spec, 
1/ we don't need to extend `WideOrgConfig` and even `WideRepoConfig` @rondDev . we could just pick up the lowest price labels  from the weight and put the the newly created issue.
2/ No need to add `default-labels` to `config`. we just need a method to pick up both `time` and `priority` labels for the lowest price

**Comment by rondDev** [comment]
Yeah, I couldn't figure out where it handles no label, I might be stupid or the naming might be a little unclear. I was going to attempt this, but code quality will be better if someone else does.

**Comment by pavlovcik** [comment]
@0xcodercrane do you have advice for @rondDev

**Comment by 0xcodercrane** [comment]
we're always welcome for any contributions and looking forward the better architecture from @rondDev .

**Comment by wannacfuture** [comment]
seems, issues.created webhook is not defined yet. 
so, should I add issueCreatedCallback to catch the issuecreate event?
